docs/setup.md

```diff
@@ -45,4 +45,4 @@ $ yarn build-prod
 
 ## Setting up NGINX
 
-Follow the setup instructions in [`/nginx/README.md`](../nginx/README.md).
\ No newline at end of file
+Follow the setup instructions in [`/nginx/README.md`](../nginx/README.md).
```

Comment:
It's possible I'm being really dim but... what's actually changed in this line?

Reply:
no idea! Didn't even think I'd changed that line...